sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/FileScanRDD.scala

```diff
@@ -45,7 +45,7 @@ import org.apache.spark.util.NextIterator
  * @param filePath URI of the file to read
  * @param start the beginning offset (in bytes) of the block.
  * @param length number of bytes to read.
- * @param modificationTime The modification time of the input file, in milliseconds.
+ * @param modificationTime The modification time of the input file, in microseconds.
```

Comment:
nit: I think we can still put `milliseconds` here, as it matches `file.getModificationTime`. We can ` * 1000` in FileScanRDD when we set the value to the internal row.

Reply:
sure! done